docs/android/platform/maps-and-location/location.md

```diff
@@ -426,18 +426,18 @@ else
 ```
 
 > [!NOTE]
->  If the user has disabled all location providers,
-`GetBestProvider` will return `null`. To see how this code works on a
-real device, be sure to enable GPS, Wi-Fi, and cellular networks under
-**Google Settings > Location > Mode** as shown in this screenshot:
-
-[![Settings Location Mode screen on an Android phone](location-images/location-02.png)](location-images/location-02.png#lightbox)
-
-The screenshot below demonstrates the location application running using `GetBestProvider`:
-
-[![GetBestProvider app displaying latitude, longitude, and provider](location-images/location-03.png)](location-images/location-03.png#lightbox)
-
-Keep in mind that `GetBestProvider` does not change the provider dynamically. Rather, it determines the best available provider once during the Activity lifecycle. If the provider status changes after it has been set, the application will require additional code in the `ILocationListener` methods &ndash; `OnProviderEnabled`, `OnProviderDisabled`, and `OnStatusChanged` &ndash; to handle every possibility related to the provider switch.
+> If the user has disabled all location providers,
+> `GetBestProvider` will return `null`. To see how this code works on a
+> real device, be sure to enable GPS, Wi-Fi, and cellular networks under
+> **Google Settings > Location > Mode** as shown in this screenshot:
+>
```

Comment:
The [current rendering](https://docs.microsoft.com/en-us/xamarin/android/platform/maps-and-location/location) does not include all these lines in the blockquote

![Screen Shot 2019-09-03 at 10 34 51 AM](https://user-images.githubusercontent.com/199373/64195508-82d40680-ce36-11e9-9375-10be3abefc2f.png)

Reply:
Yeah, I added it because the line ended in a colon

Reply:
That's fair - my general review philosophy has been to preserve the current rendering when only markdown changes are made. I'll accept the change since you are likely correct about the author's original intention.

Reply:
This one might have run long though, if you have an idea that it should stop after the first image

Reply:
Possibly? Either way, seems like it needs a manual tweak and is not easily automated. I'll merge once the other is fixed, and whatever we do with this, I'll get an Android PM to review.